forum/models/abstract.py

```diff
@@ -10,37 +10,49 @@
 class AbstractModel(ABC):
```

Comment:
Also, can we change the name of this model to MongoModel or something else instead of the AbstractModel? AbstractModel is a general name.

Reply:
updated to `MongoBaseModel`